CONTRIBUTING.md

```diff
@@ -53,19 +53,10 @@ We accept github pull requests.
 We request you give quality assurance some consideration by:
 
 * Adding go unit tests for changes where it makes sense.
-* Enabling [Travis CI](https://travis-ci.org/01org/tfortools) on your github fork of Templates for Tools to get continuous integration feedback on your dev/test branches.
+* Enabling [Travis CI](https://travis-ci.org/intel/tfortools) on your github fork of Templates for Tools to get continuous integration feedback on your dev/test branches.
 
 ## Issue tracking
 
-If you have a problem, please let us know.  IRC #ciao-project on [Freenode](https://freenode.net/kb/answer/chat)
-is a perfectly fine place
-to quickly informally bring something up, if you get a response.  The
-[mailing list](https://lists.clearlinux.org/mailman/listinfo/ciao-devel)
-is a more durable communication channel.
-
-If it's a bug not already documented, by all means please [open an
-issue in github](https://github.com/01org/intel/tfortools/issues/new) so we all get visibility
+If you have a problem, please let us know.  If it's a bug not already documented, by all means please [open an
+issue in github](https://github.com/intel/tfortools/issues/new) so we all get visibility
```

Comment:
visibility _into_ (but that whole paragraph is a bit pants.) Change it to "If you find a bug please open a github issue (url)"